.github/mergeable.yml

```diff
@@ -8,6 +8,6 @@ mergeable:
     validate:
       - do: description
         must_include:
-          regex: '((?<!([A-Z]{1,10})-?)[A-Z]+-\d+)'
+          regex: '((?<!([A-Z]{1,10})-?)[A-Z]+\d?-\d+)'
```

Comment:
```suggestion
          regex: '([A-Z]{2,10})+\d?-\d+'
```

Reply:
the lookbehind is for not matching `ABC-123-456`

Reply:
![Screenshot 2023-02-17 at 16 47 22](https://user-images.githubusercontent.com/4043765/219700552-c3889f16-0d69-46db-959e-37dd3f7e33b3.png)
![Screenshot 2023-02-17 at 16 47 10](https://user-images.githubusercontent.com/4043765/219700558-a6cb31b8-1e6c-48dd-82da-53520622bce1.png)

Looks the same no?

Reply:
Anyways, I'll approve and merge for now will see if needs fixing later.

Reply:
you're right. might be that the lookbehind is not supported anymore